examples/ansible/code/Synchronize_DNS_Zones

```diff
@@ -0,0 +1,109 @@
+- name: Synchronize DNS
+  hosts: 24-bind-01
+  vars:
+    netbox_url: 'https://<netbox-url>/'
+    netbox_token: '<redacted>'
+  tasks:
+
+#set Variables
+    - name: get active Zone Names
+      ansible.builtin.set_fact:
+        zone_names: "{{ query('netbox.netbox.nb_lookup', 'zones', plugin='netbox_dns',
+                           api_endpoint=netbox_url,
+                           api_filter='active=true',
+                           token=netbox_token)
+                    | map(attribute='value') | map(attribute='name') }}"
+
+    - name: get used Views
+      ansible.builtin.set_fact:
+        used_views: "{{ query('netbox.netbox.nb_lookup', 'views', plugin='netbox_dns',
+                           api_endpoint=netbox_url,
+                           token=netbox_token)
+                    | map(attribute='value')| map(attribute='name') }}"
+
+#Create View-Folder and remove unwanted Files
+    - name: Create View-Directory
+      ansible.builtin.file:
+        path: "/path_to_your_zone_files/{{ item }}"
+        state: directory
+        mode: '755'
+      loop: "{{ used_views }}"
+
+    - name: Collect Directory contents
+      ansible.builtin.shell: >
```

Comment:
Generally in Ansible, using `ansible.builtin.shell` is discouraged because of its security implications (and it can be a bit unpredictable, too). If possible, use `command` (which doesn't work with pipes and redirections, though), or replace it with other mechanisms that are more Ansible-like.

What you're looking for here is `ansible.builtin.find`.